libc/test/src/stdlib/rand_test.cpp

```diff
@@ -14,11 +14,18 @@
 #include <stdlib.h>
 
 TEST(LlvmLibcRandTest, UnsetSeed) {
+  static int vals[1000];
+
   for (size_t i = 0; i < 1000; ++i) {
     int val = __llvm_libc::rand();
     ASSERT_GE(val, 0);
     ASSERT_LE(val, RAND_MAX);
+    vals[i] = val;
   }
+
+  __llvm_libc::srand(1);
```

Comment:
Can you add comment about the C standard from `rand_util.cpp` here to explain why we are testing this value?